alf/utils/common.py

```diff
@@ -708,8 +708,8 @@ def get_states_shape():
         Returns 0 if internal states is not part of observation.
     """
     assert _env, "set a global env by `set_global_env` before using the function"
-    assert isinstance(_env.observation_spec(), dict), "observation not a dict"
-    if 'states' in _env.observation_spec():
+    if isinstance(_env.observation_spec(),
```

Comment:
should raise if states is not part of the observation?

Reply:
Good point.  Done.